emmaa/model_tests.py

```diff
@@ -910,11 +915,14 @@ def run_model_tests_from_s3(model_name, test_corpus='large_corpus_tests',
         test_connector = RefinementTestConnector()
     tm.make_tests(test_connector)
     filter_func = None
+    edge_filter = None
     if mm.model.test_config.get('filters'):
         filter_func_name = mm.model.test_config['filters'].get(test_corpus)
         if filter_func_name:
             filter_func = filter_functions.get(filter_func_name)
-    tm.run_tests(filter_func)
+    if mm.model.test_config.get('edge_filters'):
+        edge_filter = mm.model.test_config['edge_filters'].get(test_corpus)
```

Comment:
Related to my comment on https://github.com/sorgerlab/indra/pull/1273, we could consider implementing the edge filter function registration and loading by name around this part of the code.